boards/Kconfig

```diff
@@ -1947,6 +1947,12 @@ config ARCH_BOARD_JH7110_STAR64
 		This options selects support for NuttX on PINE64 Star64 based
 		on StarFive JH7110 SoC.
 
+config ARCH_BOARD_K230_CANMV
```

Comment:
ditto

Reply:
@xiaoxiang781216 I pushed updates for about `space-vs-tab` formating issues. please let me know if you have more feedbacks!